astropy/io/fits/tests/test_table.py

```diff
@@ -298,6 +298,19 @@ def test_ascii_table(self):
         hdul = fits.open(self.temp('toto.fits'))
         assert comparerecords(hdu.data, hdul[1].data)
         hdul.close()
+
+        # Test Scaling
+
+        r1 = np.array([11., 12.])
+        c2 = fits.Column(name='def', format='D', array=r1, bscale=2.3,
+                         bzero=0.6)
+        hdu = fits.TableHDU.from_columns([c2])
+        hdu.writeto(self.temp('toto.fits'), overwrite=True)
+        with open(self.temp('toto.fits')) as f:
+            assert '4.95652173913043548D+00' in f.read()
```

Comment:
The only real test I can do is this, checking that in the written file it correctly uses 'D'.

Reply:
Is the choice of values deliberate to not give a round number here? I'm worried that the number may differ slightly on different platforms. Is there a way to constrain the precision?

Reply:
I think the choice is deliberate, I took this from the checksum test above (`test_ascii_table_data`). And if I remember correctly from the older PR about forcing double when reading, the format is well-defined, it should give always the same value.